lib/clients/authenticated.js

```diff
@@ -111,17 +111,24 @@ _.assign(AuthenticatedClient.prototype, new function() {
   prototype._placeOrder = function(params, callback) {
     var self = this;
 
-    var requiredParams = ['size', 'side', 'product_id'];
+    var requiredParams = ['side', 'product_id'];
 
     if (params.type !== 'market') {
       requiredParams.push('price');
+      requiredParams.push('size');
```

Comment:
What happens when the params type is market and size is passed into the params anyways?